test/mocks/objects/notify-link-to-file.js

```diff
@@ -7,7 +7,8 @@ const defaultObject = {
 
 const confirmEmail = {
```

Comment:
You've coupled confirm email and retention period together meaning we can't easily test 1 w/out the other.

I would suggest creating a sep. retentionPeriod obj sim. to this and then another which brings both of them together and import _that_ into the test file.